applications/ShapeOptimizationApplication/python_scripts/response_functions/face_angle.py

```diff
@@ -112,7 +132,7 @@ def GetValue(self):
 
     def GetNodalGradient(self, variable):
         if variable != KM.SHAPE_SENSITIVITY:
-            raise RuntimeError("GetNodalGradient: No gradient for {}!".format(variable.Name))
+            raise RuntimeError(f"GetNodalGradient: No gradient for {variable.Name}!")
```

Comment:
careful. f-strings are python 3.6!

Reply:
Why I have been mentioned?

Reply:
- about f-strings: :+1: will switch to the old style
- @loumalouomega: You beeing mentioned was not related to the f-strings. Do you know if there is a utility, that assigns those conditions from modelpart A to modelpart B only if all nodes of the condition are included in modelpart B?